zookeeper-server/src/main/java/org/apache/zookeeper/ZooDefs.java

```diff
@@ -32,6 +32,10 @@ public class ZooDefs {
 
     public static final String ZOOKEEPER_NODE_SUBTREE = "/zookeeper/";
 
+    /** Root path for storing client URI-domain mappings for X509ZNodeGroupAclProvider. For more information
+     * about what info is stored under this path please see ZkClientUriDomainMappingHelper documentation. */
+    public static final String URI_DOMAIN_MAP_NODE = "/zookeeper/uri-domain-map";
```

Comment:
Let's not hard code it, this is one of the configs that we read in https://github.com/linkedin/zookeeper/blob/262b35635cb4fad0d70d70a2e75550fe845f9fd4/zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/X509AuthenticationConfig.java#L132. Or do you mean that we should strictly define this path, to make sure that this is a child node of /zookeeper?

Reply:
I think there's value in making this a nested path under `/zookeeper` because that way this will be exempt from digest calculations. Now, we might want to think about the fact that the URI-domain mapping path is configurable - should we modify our assumptions and make it always a sub-path of `/zookeeper`?